source/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -78,29 +111,67 @@ int main()
     {
         Zivid::Application zivid;
 
-        // define (picking) point in camera frame
-        const Eigen::Vector4f pointInCameraFrame(81.2F, 18.0F, 594.6F, 1.0F);
-        std::cout << "Point coordinates in camera frame: " << pointInCameraFrame.x() << " " << pointInCameraFrame.y()
-                  << " " << pointInCameraFrame.z() << std::endl;
-
         std::cout << "Reading camera pose in end-effector frame (result of eye-in-hand calibration)" << std::endl;
         const cv::Mat eyeInHandTransformation =
             readTransform(std::string(ZIVID_SAMPLE_DATA_DIR) + "/EyeInHandTransform.yaml");
 
         std::cout << "Reading end-effector pose in robot base frame" << std::endl;
         const cv::Mat endEffectorPose = readTransform(std::string(ZIVID_SAMPLE_DATA_DIR) + "/RobotTransform.yaml");
 
-        std::cout << "Converting to Eigen matrices for easier computation" << std::endl;
-        const Eigen::MatrixXf transformEndEffectorToCamera = cvToEigen(eyeInHandTransformation);
-        const Eigen::MatrixXf transformBaseToEndEffector = cvToEigen(endEffectorPose);
+        const auto dataFile = std::string(ZIVID_SAMPLE_DATA_DIR) + "/ZividGem.zdf";
+        std::cout << "Reading ZDF frame from file: " << dataFile << std::endl;
+        const Zivid::Frame frame = Zivid::Frame(dataFile);
+        auto pointCloud = frame.pointCloud();
 
         std::cout << "Computing camera pose in robot base frame" << std::endl;
-        const Eigen::MatrixXf transformBaseToCamera = transformBaseToEndEffector * transformEndEffectorToCamera;
+        const cv::Mat transformBaseToCameraCV = endEffectorPose * eyeInHandTransformation;
+
+        switch(enterCommand())
+        {
+            case CommandType::cmdTransformSinglePoint:
+            {
+                std::cout << "Transforming single point" << std::endl;
+
+                // define (picking) point as image coordinates in camera frame
```

Comment:
Capital D: define -> Define. Also, comment that coordinates are hard-coded to match the ZDF file.

Reply:
I changed my mind. It would be better if it was just written stg like: // The (picking) point is defined as image coordinates in camera frame. It is hard-coded for the ZividGem.zdf

Reply:
Done